src/components/Users/ManageUsers.tsx

```diff
@@ -105,6 +105,12 @@ export default function ManageUsers() {
   if (userListLoading || districtDataLoading || !userListData?.results) {
     return <Loading />;
   }
+  const handleTabChange = (tab: number) => {
+    setActiveTab(tab);
+    const newView = tab === 1 ? "list" : "card";
+    localStorage.setItem("usersDefaultView", newView);
+    navigate(`/users/${newView}`);
```

Comment:
setActiveTab should take care of changing the view on tab change 🤔  So we shouldn't need the navigate here.

Reply:
Yep done